canonical-websites/maas.io
# Migrate maas.io overview to VBT

## Done

- Use VBT
- New VF header and footer
- Change old vanilla variables
- Add new color settings file and variables
- Remove unneeded SASS files
- Change asset path in HTML
- Add site specific SASS
- Replace overview HTML classes with new VBT and VF classes

## QA

- ./run then pop to http://0.0.0.0:8006/
- Make sure everything looks okay

NB: ticks and external icon images are WIP

## Issue / Card

Fixes #146 

## Demo
http://maas.io-vbt-overview.demo.haus
